include/wx/generic/gridsel.h

```diff
@@ -114,6 +117,11 @@ class WXDLLIMPEXP_CORE wxGridSelection
     void EndSelecting();
     void CancelSelecting();
 
+    // Return the PolyPolygon object. Call ComputePolyPolygon() if necessary.
```

Comment:
This comment should probably mention that it returns the intersection of the shape of the selection with the given rectangle (if I understand what it does correctly).